js/src/views/Application/Status/status.js

```diff
@@ -68,9 +69,14 @@ class Status extends Component {
       return null;
     }
 
+    const [protocol, rest] = enode.split('://');
+    const [id, host] = rest.split('@');
+    const abbreviated = `${protocol}://${id.slice(0, 3)}…${id.slice(-3)}@${host}`;
```

Comment:
Definately better. Looking at it now, we should probably just move it completely to the status page and not have it here at all - as you suggested earlier, it is not really user-facing.